docs/websites/varnish/use-varnish-with-nginx-to-serve-cached-wordpress-content-over-ssl-on-debian-8.md

```diff
@@ -310,6 +309,12 @@ For all steps in this section, replace `203.0.113.100` with your Linodes public
 
         sudo systemctl daemon-reload
 
+3.  To make Varnish start at boot up, run the following command:
```

Comment:
I'm pretty sure it's enabled on install, as is the case with most services on Debian based systems.